components/molecule/fieldTextarea/README.md

```diff
@@ -0,0 +1,93 @@
+# MoleculeFieldTextarea
+
+
+`MoleculeFieldTextarea` is a component that wraps a composition of Label + Textarea + Validations  Messages. 
+
+
+## Installation
+
+```sh
+$ npm install @s-ui/react-molecule-field-textarea --save
+```
+
+## Usage
+
+After importing the component `MoleculeFieldTextarea` like this
+
+```javascript
+import MoleculeFieldTextarea from '@s-ui/react-molecule-field-textarea'
+```
+
+### Basic usage
+    
+```
+  <MoleculeFieldTextarea
+    id="description"
+    label="Description"
+  />
+```
+
+### With initial value
+    
+```
+  <MoleculeFieldTextarea
+    id="description"
+    label="Description"
+    value="In some place of La Mancha which name..."
+  />
+```
+
+### With maxCharacters
+    
+```
+  <MoleculeFieldTextarea
+    id="description"
+    label="Description"
+    value="In some place of La Mancha which name..."
+    maxCharacters={75}
```

Comment:
the shorter the API the better as long as it's easily understandable. IMO `maxChars` would be a better name.

Reply:
👍 changed to `maxChars`